### DISCUSSION TITLE
Binary to Text <-> Text to Binary Converter

### DISCUSSION THREAD

**Discussion by osmanzeki** [DevToys-app/DevToys#622] [Ideas]
Sometimes you have a little string of binary that you want to convert to text and back.

Sometimes, game designers like to use binary as a tech puzzle in their games and you wanna be able to quickly look it up without having to write a quick script for it. :)

I'm not sure how useful it would be to everyone here, but I figured it should be relatively simple to implement and might be cool to have among the other useful utilities.

**Comment by jamescurran** [comment]
Convert to/from Text using what encoding?  ASCII? UTF-8? UTF-16?  It get complicated.

**Reply by groblerf** [reply]
Why not add multiple, so the user can select?
Options indicating binary type such as Big-endian or Little-endian, and string value type expected, such as LPString, LPUTF8Str, LPWString, AnsiBStr, BStr, TBstr, VBByRefStr, and Character set. This way all types null terminated or not, with or without length bytes are covered, even single byte C strings.
